tests/deb_rpm_tar/BUILD

```diff
@@ -0,0 +1,65 @@
+load("//pkg:deb.bzl", "pkg_deb")
+load("//pkg:tar.bzl", "pkg_tar")
+load("//pkg:rpm.bzl", "pkg_rpm")
+load("//pkg:mappings.bzl", "pkg_files")
+
+genrule(
+    name = "gen_foo",
+    outs = ["usr/bin/foo"],
+    cmd = "touch $@",
+)
+
+pkg_tar(
+    name = "deb_input",
+    srcs = [":gen_foo"],
+)
+
+pkg_deb(
+    name = "test_deb",
+    data = ":deb_input",
+    description = "Some package",
+    maintainer = "john.doe@example.com",
+    package = "debpackage",
+    package_file_name = "debpackage-0.0.0.deb",
+    version = "0.0.0",
+)
+
+pkg_tar(
```

Comment:
Why would someone do this?

Reply:
What if I have to ship multiple files, i.e. instructions or deployment scripts, along with deb/rpm that cannot be part of the package to the air gapped environment? The tarball is the easiest way to distribute.

Reply:
OK. 
But we don't need any of this. There are existing test targets that exhibit the right characteristics. Or, if not, they should be in the respective test/rpm or test/deb trees.